code/datums/ammo/ammo.dm

```diff
@@ -173,6 +173,10 @@
 		living_mob.apply_stamina_damage(fired_projectile.ammo.damage, fired_projectile.def_zone, ARMOR_BULLET)
 
 /datum/ammo/proc/slowdown(mob/living/living_mob, obj/projectile/fired_projectile)
+	if(isxeno(living_mob))
+		var/mob/living/carbon/xenomorph/xeno = living_mob
+		if(xeno.caste.tier > 2)
+			return //tier 3 are not affected
```

Comment:
This should work but you likely don't want queen to be affected either so needs additional checks.

Reply:
Feel free to tweak slugs too if you don't like them behaving different.